bukuserver-runner/buku-server.py

```diff
@@ -224,21 +230,21 @@ def prepare_vars():
     if exec and os.path.isdir(os.path.expanduser(exec)):
         workdir, exec = exec, os.path.join('bukuserver', 'server.py')
     return {
-        'confdir': confdir,
+        'dbdir': os.path.abspath(os.path.expanduser(env.get('BUKU_DEFAULT_DBDIR') or confdir)),
```

Comment:
Renamed for clarity; unlike `confdir`, this parameter is only used for DB location (which is why `BUKU_DEFAULT_DBDIR` is handled here)